python/lsst/ts/xml/data/sal_interfaces/SALSubsystems.xml

```diff
@@ -171,9 +171,9 @@
     <ActiveDevelopers>Petr Kubanek</ActiveDevelopers>
     <Github>https://github.com/lsst-ts/ts_dream</Github>
     <JenkinsTestResults>Not Available</JenkinsTestResults>
-    <RubinObsContact>Patrick Ingraham</RubinObsContact>
+    <RubinObsContact>Parker Fagrelius</RubinObsContact>
```

Comment:
Sorry Rob, I think I told you the wrong person here, Brian Stalder would be the person taking over the DREAM camera, not Parker